### PR TITLE
docs: update CITATION

### LINKED ISSUE #61: [DOCS] Add information to the CITATION file
In reference with #25.

### PR DESCRIPTION
**Pull request by amangoel185**
I have updated the CITATION with information from the Matplotlib Training Presentation. My apologies if I got something wrong. A few more changes need to be made before the CITATION is finalised:

- [x] Add abstract. We need a one liner for the module.
- [x] Add ORCID for everyone if available.
- [x] Add/Remove License. Since `CC BY 4.0` license is used after modification, do we mention `CC BY 4.0` license or nothing at all?

In reference with #61.

### CODE REVIEW

**Comment by klieret** [comment]
Regarding the license, it's still a CCBY4.0, the modification is only regarding the copyright owners, I think

File: CITATION [hunk 1]
```diff
@@ -1 +1,30 @@
...
+  given-names: "Meirin"
+  orcid: "FIXME"
+  affiliation: "University of Sussex"
+- family-names: "García Hernández"
+  given-names: "Marcela"
+  orcid: "FIXME"
+  affiliation: "Center for Research and Advanced Studies of the National Polytechnic Institute"
+- family-names: "Antonio Fidalgo-Rodríguez"
+  given-names: "Guillermo"
+  orcid: "FIXME"
```
Comment (GuillermoFidalgo) [inline CITATION]: Add 0000-0001-8605-9772 as orcid Id number.

**Review by GuillermoFidalgo — CHANGES_REQUESTED** [review]
Add 0000-0001-8605-9772 as my Orcid Id number.

**Review by GuillermoFidalgo — APPROVED** [review]

**Review by klieret — APPROVED** [review]
LGTM!

File: CITATION [hunk 2]
```diff
@@ -1 +1,30 @@
-FIXME: describe how to cite this lesson.
\ No newline at end of file
+cff-version: 1.2.0
+message: "Please cite the following works when using this software."
+type: software
+title: "Matplotlib for HEP"
+abstract: "FIXME"
```
Comment (klieret) [inline CITATION]: Still one fixme here though. Just copy paste the thing from the index

**Review by klieret — CHANGES_REQUESTED** [review]

**Comment by amorenobr** [comment]
My Orcid ID number is 0000-0001-8415-2543

**Review by klieret — APPROVED** [review]
Thanks a lot!

**Comment by Marchela04** [comment]
Thank you @amangoel185 
I didn't have an id, but I created one
https://orcid.org/0000-0003-2393-3367

**Pull request merged**